### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Gaphor/gaphor.github.io](https://hosted.weblate.org/projects/gaphor/gaphor-github-io/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/site.hr.po [hunk 1]
```diff
@@ -58,7 +58,11 @@ msgstr "Gaphor je UML, SYSML, Raaml i C4 program za modeliranje. Izrađen je kao
 #. type: Plain text
 #: _includes/content.md.filt
 msgid "Build Class, Interaction, and State Machine diagrams for software or Block Definition and Requirements diagrams for systems. Model the elements you need. If you want to mix and match, you can even add different diagram items to the same diagram to get the view you need."
-msgstr "Izradi klase, interakciju i dijagrame stanja uređaja za softver ili dijagrame definicija blokova i dijagrame zahtjeva za sustave. Modeliraj elemente koje trebaš. Ako želiš miješati i uskladiti, za dobivanje željenog prikaza istom dijagramu možeš čak dodati različite elemente dijagrama."
+msgstr ""
+"Izradi klase, interakciju i dijagrame stanja uređaja za softver ili "
+"dijagrame definicija blokova i dijagrame preduvjeta za sustave. Modeliraj "
+"elemente koje trebaš. Ako želiš miješati i uskladiti, za dobivanje željenog "
+"prikaza istom dijagramu možeš čak dodati različite elemente dijagrama."
```
Comment (danyeaw) [inline po/site.hr.po:65]: @milotype Do you know why this is on multiple lines like this now?

Comment (milotype) [reply po/site.hr.po:65]: No, sorry, I don't know. BTW, I'm not a programmer, but:

- when I look at https://hosted.weblate.org/translate/gaphor/gaphor-github-io/hr/?checksum=2a3ffa7f29da1e00&q=Izradi+klase&sort_by=-priority%2Cposition there are no line breaks.
- when I look into the current file [site.hr.po](https://github.com/gaphor/gaphor.github.io/blob/main/po/site.hr.po) withing gaphor.github.io GitHub page there are no line breaks.
- when I download the file from https://hosted.weblate.org/projects/gaphor/gaphor-github-io/hr/ and open it with **Poedit**, there are no line breaks. **But** when I open that file with **BBEdit** then I see the same result as you.

Strange!

Comment (milotype) [reply po/site.hr.po:65]: Also, when I copy the string from https://hosted.weblate.org/translate/gaphor/gaphor-github-io/hr/?checksum=2a3ffa7f29da1e00&q=Izradi+klase&sort_by=-priority%2Cposition and paste it into TextEdit (MacOS) as plain text, there are also no line breaks.

Comment (danyeaw) [reply po/site.hr.po:65]: Interesting, I'm not sure either!

**Pull request merged**